<launch>
  <arg name="host" default="wallarelvo-tower.csail.mit.edu" />
  <arg name="port" default="8080"/>
  
  <node pkg="baxter_remote_control" name ="control_h" type = "BaxterInterface.py" output="screen">
    <param name="part" value="head" />
  </node>
  <node pkg="baxter_remote_control" name ="control_r" type = "BaxterInterface.py" output="screen">
    <param name="part" value="right" />
  </node>
  <node pkg="baxter_remote_control" name ="control_l" type = "BaxterInterface.py" output="screen">
    <param name="part" value="left" />
  </node>
  <node pkg="baxter_remote_control" name ="control_l_g" type = "BaxterInterface.py" output="screen">
    <param name="part" value="left_gripper" />
  </node>
  <!--<node pkg="baxter_remote_control" name ="control_r_g" type = "BaxterInterface.py" output="screen">
    <param name="part" value="right_trigger" />
  </node> -->

  <node pkg="canopy_client" type="client_node.py" name="canopy_client" output="screen">
    <param name="name" value="Baxter"/>
    <param name="host" value="$(arg host)"/>
    <param name="port" value="$(arg port)"/>
    <param name="private_key" value="4wsDxqdQpXygyj4t4" />
    <param name="description" value="baxter side" />
    <rosparam>
      publishing:
      - /left_ros_currentpos
      - /left_ros_valid
      - /robot/range/left_hand_range/state
      - /right_ros_currentpos
      - /right_ros_valid
      - /robot/range/right_hand_range/state

      types:
      - geometry_msgs/Pose
      - std_msgs/Bool
      - sensor_msgs/Range
      - geometry_msgs/Pose
      - std_msgs/Bool
      - sensor_msgs/Range


      trusted:
      - ".*"
      - ".*"
      - ".*"
      - ".*"
      - ".*"
      - ".*"
    </rosparam> 
  </node>


</launch>
